src/common/services/api/order.service.js

```diff
@@ -407,7 +401,7 @@ class Order {
 
   calculatePriceWithoutFees (originalAmount) {
     originalAmount = parseFloat(originalAmount)
-    const newAmount = originalAmount / this.removeFeesDividend
+    const newAmount = originalAmount - (originalAmount * 0.0235)
```

Comment:
You've got 2 different versions of this. 
```suggestion
    const newAmount = originalAmount * 0.9765
```
would be the inverse of the equation you're using above in `calculateAmountWithFees`. I'm fine with using the original equation (`amt_gift     = amt_with_fee - amt_with_fee * 0.0235`) but you're using the reduced version above.